sllm/serve/store_manager.py

```diff
@@ -372,9 +372,17 @@ async def register(self, model_config):
             for node_id, node_info in worker_node_info.items():
                 node_address = node_info["address"]
                 if node_id not in self.local_servers:
-                    self.local_servers[node_id] = SllmLocalStore(
-                        node_id, SllmStoreClient(f"{node_address}:8073"), 1
-                    )
+                    if self.local_servers:
```

Comment:
Is this `if` check necessary? The previous line (L374) already uses `if node_id not in self.local_servers:`. If `local_servers` doesn't exist, an error would occur before this `if` statement anyway. What do you think?

Reply:
I see, the change here is not clear, to clarify my intent of making this change: when a new `SllmLocalStore` was created within this block, since `SllmLocalStore` needs 5 args but only 3 were given, it raised error. So I checked if `local_servers` contains any existing entries, if yes, I can copy the same chunksize and hardware info from the existing nodes.
This happened when I tried with multiple workers on the same node (later I realized we assume one worker id per node).

Reply:
I see.